.gitignore

```
@@ -360,4 +359,4 @@ MigrationBackup/
 .ionide/
 
```

Make sure the file ends with a newline character. This helps avoid issues with some tools and adheres to POSIX standards.

**Suggested change**
```suggestion
FodyWeavers.xsd

```